utils/convert_srg_export_to_xlsx.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import datetime
+import os
+import openpyxl
+from openpyxl.styles import Alignment, Font
+
+
+SSG_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+RULES_JSON = os.path.join(SSG_ROOT, "build", "rule_dirs.json")
+BUILD_CONFIG = os.path.join(SSG_ROOT, "build", "build_config.yml")
+OUTPUT = os.path.join(SSG_ROOT, 'build',
+                      f'{datetime.datetime.now().strftime("%s")}_stig_export.xlsx')
+
+
+def setup_sheet(sheet: openpyxl.worksheet.worksheet.Worksheet) -> None:
+    sheet.column_dimensions['A'].width = 17
+    sheet.column_dimensions['B'].width = 17
+    sheet.column_dimensions['C'].width = 17
+    sheet.column_dimensions['E'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['F'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['G'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['H'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['I'].width = 1.5 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['K'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['J'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['M'].width = 4 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['O'].width = 3 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['P'].width = 3 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['Q'].width = 3 * sheet.column_dimensions['A'].width
+
+
+def setup_headers(sheet: openpyxl.worksheet.worksheet.Worksheet) -> None:
+    sheet['A1'] = 'IA Control'
+    sheet['B1'] = 'CCI'
+    sheet['C1'] = 'SRGID'
+    sheet['D1'] = 'STIGID'
+    sheet['E1'] = 'SRG Requirement'
+    sheet['F1'] = 'Requirement'
+    sheet['G1'] = 'SRG VulDiscussion'
+    sheet['H1'] = 'Vul Discussion'
+    sheet['I1'] = 'Status'
+    sheet['J1'] = 'SRG Check'
+    sheet['K1'] = 'Check'
+    sheet['L1'] = 'SRG Fix'
+    sheet['M1'] = 'Fix'
+    sheet['N1'] = 'Severity'
+    sheet['O1'] = 'Mitigation'
+    sheet['P1'] = 'Artifact Description'
+    sheet['Q1'] = 'Status Justification'
+    for cell in list(sheet.iter_rows(max_row=1))[0]:
+        cell.font = Font(bold=True, name='Calibri')
+
+
+def format_cells(sheet: openpyxl.worksheet.worksheet.Worksheet, row_num: int):
+    for row in sheet.iter_rows():
+        for cell in row:
+            cell.alignment = cell.alignment = Alignment(wrap_text=True, vertical='top')
+
+
+def setup_row(sheet: openpyxl.worksheet.worksheet.Worksheet, row: dict, row_num: int) -> None:
+    sheet[f'A{row_num}'] = row['IA Control']
+    sheet[f'B{row_num}'] = row['CCI']
+    sheet[f'C{row_num}'] = row['SRGID']
+    sheet[f'D{row_num}'] = row['STIGID']
```

Comment:
```suggestion
    sheet[f'D{row_num}'] = row.get('STIGID', '')
```

In my tests it complained row didn't have `STIGID`. Maybe we should reconsider how this is populated. Using `get` is one of the solutions.

Reply:
I have rebased on master, which now has this field in the export.